third_party/rust/crates/Cargo.raze.lock

```diff
@@ -102,7 +102,7 @@ dependencies = [
  "cfg-if 1.0.0",
  "libc",
  "miniz_oxide",
- "object 0.28.3",
+ "object 0.28.4",
```

Comment:
Why did the versions get bumped in the lock file? (Every time these get bumped it requires updating the air-gapped cache.)

Reply:
This will keep happening unless we pin exact versions of everything in the main Cargo.toml. We should probably do this regardless but if it's creating a problem for airgapping we should fix it sooner rather than later.

(It's kind of orthogonal to this change so resolving.)

Reply:
Ah got it, yeah no problem. I can pin them in a follow up.